translations/de.log

```diff
@@ -0,0 +1,97 @@
+*****: de
+orig->languageName: 'en', 'de'
+orig->control: 'Control', 'Steuerung'
+orig->print: 'Print', 'Druck'
+orig->status: 'Status', 'Status'
+orig->console: 'Console', 'Konsole'
+orig->setup: 'Setup', 'Setup'
```

Comment:
for "Setup" the german word is "Einstellungen". 
But "Setup" is understandable for every german speaking person and it is shorter...

Reply:
Done.

Reply:
Done.